### PR TITLE
Clean up poms

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

 *  fix scm deprecation errors
 * remove duplicate dependencies

## Related issues

related to https://github.com/zeebe-io/zeebe/issues/2165

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to backport the fix to the last two minor versions

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the release announcement

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -1070,7 +1054,7 @@
               </goals>
               <configuration>
                 <rules>
-                  <dependencyConvergence />
+                  <dependencyConvergence/>
                 </rules>
               </configuration>
             </execution>
```
Comment (npepinpe) [inline parent/pom.xml:1060]:
```suggestion
            </execution>
             <execution>
              <id>enforce-unique-dependencies</id>
              <goals>
                <goal>enforce</goal>
              </goals>
              <configuration>
                <rules>
                  <banDuplicatePomDependencyVersions/>
                </rules>
              </configuration>
            </execution>
```

**Review by npepinpe — CHANGES_REQUESTED** [review]
Thanks for this! Only one suggestion, I would enforce not having duplicate dependencies. wdyt?

**Comment by Zelldon** [comment]
Lets do it👍

**Comment by Zelldon** [comment]
Bors r+

**Review by npepinpe — APPROVED** [review]
Sorry i thought I had approved it

**Comment by Zelldon** [comment]
Me2😬

Bors r+

**Comment by npepinpe** [comment]
bors retry

**Pull request merged**